packages/e2e-tests/next-app/cypress/integration/pages.test.ts

```diff
@@ -70,6 +70,22 @@ describe("Pages Tests", () => {
         cy.visit(path);
       });
 
+      it(`supports preview mode ${path}`, () => {
```

Comment:
The assertion here is:

- We can enable preview through the API
- We can then visit a site that has a preview from SSG
- That page is a preview so should not be cached

- We can disable preview
- We can visit the page again and it has preview `false`
- This page is cached

Reply:
Nice, I think that should be complete.